### DISCUSSION TITLE
AdNauseam being detected by smaller, fringe sites.

### DISCUSSION THREAD

**Discussion by blazingwinter** [dhowe/AdNauseam#2537] [General]
It's being detected on smaller, fringe sites, like 360haven.com. It's been a hot minute since I've visited this site, and maybe they've installed a "can't use this site with adblocker" thing in those months, but things like this I've noticed are becoming more prevalent in recent times. Last time I visited this site, I didn't have any issues, and could be either a version thing, or what I mentioned before. I couldn't post this as an actual issue, because the configuration box is mandatory and I couldn't get that filled out, and didn't know what it was. 

![image](https://github.com/dhowe/AdNauseam/assets/92473553/c935fc18-f844-4982-935c-86efa823c3d9)

**Comment by dhowe** [comment]
have you visited this site with ublock origin -- is it also similarly detected ?

**Comment by mneunomne** [comment]
Could not currently reproduce. perhaps already fixed by the filter lists.
